app/service/update-appeal-service.ts

```diff
@@ -150,7 +153,7 @@ export default class UpdateAppealService {
     };
   }
 
-  private getDate(ccdDate): AppealDate {
+  getDate(ccdDate): AppealDate {
```

Comment:
Are we using getDate somewhere?

Reply:
only used within the class, can put back to private if you'd like